### PR TITLE
ingest: add common options to each processor's documentation

### PR DESCRIPTION
**Pull request by jakelandis**
* adds `if`, `on_failure`, `ignore_failure`, and `tag` to table for each processor

part of #33188

-----------

Example rendered processor:
image

EDIT : update example

### CODE REVIEW

**Comment by jakelandis** [comment]
tagged as going back to only 6.5.0 since that is the first introduction of the `if` conditional and as of 6.5.0 Kibana console will also allow auto complete these 3 options. However, on_failure and tag (i believe) are valid back to the initial release.

**Comment by jakelandis** [comment]
added `ignore_failure` to common options (missed in original commit)

File: docs/reference/ingest/ingest-node.asciidoc [hunk 1]
```diff
@@ -791,6 +792,8 @@ Accepts a single value or an array of values.
 --------------------------------------------------
 // NOTCONSOLE
 
+
+
```
Comment (rjernst) [inline docs/reference/ingest/ingest-node.asciidoc]: unintentional extra newlines?

**Review by rjernst — APPROVED** [review]
LGTM

File: docs/reference/ingest/ingest-node.asciidoc (hunk 1 above)
Comment (jakelandis) [reply docs/reference/ingest/ingest-node.asciidoc]: fixed. thanks!

**Pull request merged**